zenodo/modules/records/data/datacite-v4.1-full-example.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
Is this file (and other example files under this folder) being used by tests? If yes, it should be somewhere under the `tests` folder.